internal/services/synapse/synapse_managed_private_endpoint_resource.go

```diff
@@ -177,8 +192,23 @@ func resourceSynapseManagedPrivateEndpointDelete(d *pluginsdk.ResourceData, meta
 		return err
 	}
 	if _, err := client.Delete(ctx, id.ManagedVirtualNetworkName, id.Name); err != nil {
-		return fmt.Errorf("deleting Synapse Managed Private Endpoint %q (Workspace %q / Resource Group %q): %+v", id, id.WorkspaceName, id.ResourceGroup, err)
+		return fmt.Errorf("deleting %s: %+v", id, err)
```

Comment:
this is a parsed ID, so it'll be returned as a pointer - as such this needs to be:

```suggestion
		return fmt.Errorf("deleting %s: %+v", *id, err)
```